test/data/models/bad_configuration.rb

```diff
@@ -1,5 +1,3 @@
-ActiveSupport::Deprecation.silence do
```

Comment:
I think this is not needed any more. I could not see any deprecations in the tes output. If it is still needed I think it needs to be changed to `ActiveSupport::Deprecation.new.silence do`